core/src/test/java/org/elasticsearch/fieldstats/FieldStatsTests.java

```diff
@@ -144,44 +144,44 @@ private void testNumberRange(String fieldName, String fieldType, long min, long
 
     public void testMerge() {
         List<FieldStats> stats = new ArrayList<>();
-        stats.add(new FieldStats.Long(1, 1l, 1l, 1l, 1l, 1l));
-        stats.add(new FieldStats.Long(1, 1l, 1l, 1l, 1l, 1l));
-        stats.add(new FieldStats.Long(1, 1l, 1l, 1l, 1l, 1l));
```

Comment:
The great irony is that these are really hard to read in the font that chrome uses for plain text files so I have to squint to make sure these are right.....